File: PastaPHP/Model/UsuarioSistema.php
<?php 

require_once __DIR__ . "/Pessoa.php";

class UsuarioSistema extends Pessoa{
     private $email;
     private $senha;
     private $anoLetivo;
     private $cargo;

    public function __construct() {
    parent::__construct();
    }
     /* public function __construct($nome, $id, $cpf, $email, $senha) {
        parent::__construct($nome, $id, $cpf);
        $this->email = $email;
        $this->senha = $senha;
    }*/

    public function getEmail() {
        return $this->email;
    }

    public function setEmail($email) {
        $this->email = $email;
    }

    public function getSenha() {
        return $this->senha;
    }

    public function setSenha($senha) {
        $this->senha = $senha;
    }

      public function getAnoLetivo(){
        return $this->anoLetivo;
    }
    
    public function setAnoLetivo($anoLetivo){
        $this->anoLetivo = $anoLetivo; 
    }

      public function getCargo(){
        return $this->cargo;
    }
    
    public function setCargo($cargo){
        $this->cargo = $cargo; 
    }

      public function __toString(){
        return parent::__toString(). "Email: {$this->email} - Senha: *****  - Ano Letivo: {$this->anoLetivo} - Cargo: {$this->cargo}";
    }

}
?>

File: PastaJava/Model/UsuarioSistema.java
public abstract class UsuarioSistema extends Pessoa { 
    private String email;
    private String senha;

    public UsuarioSistema() {}

    public UsuarioSistema(String nome, int id, String cpf, String email, String senha) {
        super(nome, id, cpf);
        this.email = email;
        this.senha = senha;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getSenha() {
        return senha;
    }

    public void setSenha(String senha) {
        this.senha = senha;
    }

    @Override
    public String toString() {
        return super.toString() + " [Email=" + email + ", Senha=****]";
    }
}
